subiquity/common/resources.py

```diff
@@ -24,3 +24,26 @@
 
 def resource_path(relative_path):
     return os.path.join(os.environ.get("SUBIQUITY_ROOT", "."), relative_path)
+
+
+def get_users_and_groups(target, root):
+    # prevent import when calling just resource_path
+    from subiquitycore.utils import run_command
+
+    users_and_groups_path = resource_path('users-and-groups')
+    if os.path.exists(users_and_groups_path):
+        groups = open(users_and_groups_path).read().split()
+    else:
+        groups = ['admin']
+    groups.append('sudo')
+
+    command = ['chroot', target, 'getent', 'group']
```

Comment:
For instance, here, this is something that is not done on subiquity side if I’m correct, so worth a discussion with upstream subiquity. Let’s keep it aside for now.

Reply:
This is done in the subiquity side though. It's originally the function of get_target_group() that I merged for modulizing

Reply:
ack, let’s discuss that in the other PR.